src/bundle/Resources/public/scss/_main-row.scss

```diff
@@ -0,0 +1,32 @@
+.ez-main-row {
+    display: grid;
+    grid-template-columns: 1fr calculateRem(80px);
+    grid-template-rows: auto;
+    grid-template-areas: "ez-content-container ez-context-menu";
```

Comment:
CS: Please, use single quotes instead. It's applicable in all other places in the CSS/SCSS code.